_posts/2022-12-29-vanilla-os-22-10-kinetic.md

```diff
@@ -97,7 +97,7 @@ We designed this system to take away an annoying task from the user, who
 simply wants to do their own thing. Updates go through ABRoot transactions and 
 are applied on the next reboot, without taking extra time during boot.
 
-![VSO Control Center](/assets/uploads/vanilla-os-updates.png)
+![The Updates tab of the Vanilla Control Center showing the latest update check, weekly updates scheduling set, and "SmartUpdate," where the system will not update if the system is under load or the battery is low.](/assets/uploads/vanilla-os-updates.png)
```

Comment:
```suggestion
![The Updates tab of the Vanilla Control Center shows the latest update check, weekly updates scheduling set, and "SmartUpdate," where the system will not update if the system is under load or the battery is low.](/assets/uploads/vanilla-os-updates.png)
```

Reply:
"showing" works better here than "shows" imo. This is a description of the image, not of what VCC *does*. "shows" would work better in a description of the project.